local_node.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
+
 KEYS[0]="dev0"
 KEYS[1]="dev1"
 KEYS[2]="dev2"
 CHAINID="evmos_9000-1"
 MONIKER="localtestnet"
 # Remember to change to other types of keyring like 'file' in-case exposing to outside world,
-# otherwise your balance will be wiped quickly.
+# otherwise your balance will be wiped quickly
```

Comment:
Why remove the dot here?

Reply:
Intially removed to make CI do not use cached job result(previous lint run was cached and I have to make changes to break cache), anyways rest of the comments have no dots so I see no big reason to have dot here

Reply:
That's true, there aren't really dots used anywhere else too